documentation/source/building.rst

```diff
@@ -273,3 +273,19 @@ Additional Environment Variables
     Path to the directory containing the cocotb Makefiles and simulator libraries in the subdirectories
     :file:`lib`, :file:`include`, and :file:`makefiles`.
     You don't normally need to modify this.
+
+.. envvar:: GPI_EXTRA
+
+    A comma-separated list of extra libraries that are dynamically loaded at runtime.
+    A function from each of these libraries will be called as an entry point prior to elaboration, allowing these libraries to register
+    system functions and callbacks. Note that HDL objects cannot be accessed at this time.
+    The function name defaults to ``{library_name}_entry_point``, but a custom name can be specified using a ``:``, which follows an existing simulator convention.
+
+    For example:
+
+    * ``GPI_EXTRA=name`` will load ``libname.so`` with default entry point ``name_entry_point``.
+    * ``GPI_EXTRA=nameA:entryA,nameB:entryB`` will load ``libnameA.so`` with entry point ``entryA`` and ``libnameB.so`` with entry point ``entryB``.
+
+    .. versionchanged: 1.4.0
```

Comment:
```suggestion
    .. versionchanged:: 1.4.0
```

Reply:
Doesn't render without this: https://cocotb--1457.org.readthedocs.build/en/1457/building.html#envvar-GPI_EXTRA